src/lib/config.coffee

```diff
@@ -60,6 +60,10 @@ module.exports = (dnschain) ->
             internalTLSPort: 2500   # Not accessible from the internet, used internally only
             internalAdminPort: 3000 # Not accessible from the internet, used internally only
             host: '0.0.0.0' # what we bind to. 0.0.0.0 for the whole internet
+        redis:
+            enabled: false
+            host: '127.0.0.1'
+            port: 6379
```

Comment:
Let's support unix domain sockets since they're a very common usage case.

Remove `host` and `port` and replace with:

```coffeescript
socket: '127.0.0.1:6379`
```
Allow that string to also be the path to a unix domain socket.

When choosing which `createClient` function to use, simply see if the string contains a `:`. If it does, do `[host, port] = gConf.get('redis:socket').split(':')`, etc. Otherwise assume it's a path to a unix domain sock.